app.js

```diff
@@ -40,7 +40,7 @@ if (process.env.NEAR_CONTRACT_HELPER_PUBLIC_KEY && process.env.NEAR_CONTRACT_HEL
 const defaultKey = new KeyPair(publicKey, secretKey);
 const keyStore = new InMemoryKeyStore();
 keyStore.setKey(defaultSender, defaultKey);
-const localNodeConnection = new LocalNodeConnection(process.env.INTERNAL_NODE_HOST || 'http://localhost:3030');
+const localNodeConnection = new LocalNodeConnection(process.env.NODE_URL || 'https://studio.nearprotocol.com/devnet');
```

Comment:
why change default value?

Reply:
The convention across all repos is that the default values are for the main setup that runs in the cloud is under our main DNS. GCloud deployment or local deployment need to override them.